src/shared/components/HeaderTextField/HeaderTextField.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import { Container, Title, WarningText, StyledTooltip, StyledInput } from './HeaderTextField.style'
+
+type HeaderTextFieldProps = {
```

Comment:
How do you access this component's value from its consumer? You made the input self-controlling (it has its own state and handles the updates on its own). This way we won't be able to build a form with it. `value` and `onChange` should come from outside, via props